test/bdd/features/aries_router_e2e_sdk.feature

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@aries_router
+Feature: DIDComm Transport between two Edge Agents(without Inbound) through DIDComm Routers
+
+  # https://wiki.hyperledger.org/display/ARIES/DIDComm+MediatorRouter
+  Scenario: Decentralized Identifier(DID) Exchange between two Edge Agents through Routers
```

Comment:
Minor : Should use `When, then ...` instead of just `And, And.....`

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/638de37c7ee7563d11e6d71efccf8bbba444de00..e75dea7c58e3b3d0ea308f21a689395d82016bd3#diff-6c628f852b19b92aa1ffe149b1485119R20).